### ISSUE TITLE
Proposed new keyword: "hidden"

### ISSUE THREAD

**Issue by nitinpatel811** [jhipster/generator-jhipster#9163]
##### **Overview of the feature request**

This is a request to add new attribute at entity field level to define whether it will be visible on the UI or not. Not all fields defined in an Entity are supposed to be shown on the UI screen. For example, I have a field in almost all of my entities is called "deleted" which I use for soft deleting the data from database. This field is never supposed to be visible on the UI. It is ok, if it flows till UI layer as part of model, but need not be shown on UI screen.

##### **Motivation for or Use Case**

This feature will allow users to define entity fields as "hidden". 

##### **Related issues or PR**

I could not find any such related issue or PR.

-   [ x] Checking this box is mandatory (this is just to show you read everything)

**Comment by pascalgrimaud** [comment]
To hide the field in a generated code, it will take less than 1 minute.
To add this new keyword 'hidden', to test it in our CICD, to maintain it, etc... it will require a lot of work.

So for me, it is against our Policy 2: only add options when there is sufficient added-value in the generated code

Let's wait what other users think about that.

**Comment by MathieuAA** [comment]
I agree with Pascal here, there's not much value in the end for this feature

**Comment by jdubois** [comment]
But thanks for the idea & the feature request @nitinpatel811 !!
